wordpress-mobile/WordPress-iOS
# Swift App Delegate 5: Appearance

Continuing from #11556, this PR ports over the appearance customisation methods from the app delegate to Swift.

At this stage, I've kept the code as close as possible to the objective-c to make it easier to review.

**To test:**

* Review the code
* Build and run, and do a smoke test of the app check things look as they should! Check through some of the specific items customised in the code here, like the media picker.

@nheagy would you be up for reviewing this one?
